test/client/deposit.test.ts

```diff
@@ -49,12 +50,12 @@ describe("Deposit feature", () => {
 
             assert.equal(subtree1.id.toNumber(), 1);
             const subtreeStates1 = [states[0], states[1]];
-            assert.deepEqual(subtree1.states, subtreeStates1);
+            assert.deepEqual(encode(subtree1.states), encode(subtreeStates1));
             assert.equal(subtree1.root, getMerklizedRoot(subtreeStates1));
 
             assert.equal(subtree2.id.toNumber(), 2);
             const subtreeStates2 = [states[2], states[3]];
-            assert.deepEqual(subtree2.states, subtreeStates2);
+            assert.deepEqual(encode(subtree2.states), encode(subtreeStates2));
```

Comment:
Not sure how these were passing before as normal equality checks on `BigNumbers` no worky good.

Reply:
in my experience, `expect` handles BigNumber comparisons much better than `assert`

Reply:
We certainly could use `expect`s but would likely want t change to that style in all tests. Do they just have better BN parsing in that testing flow?